lib/gitlab/client/projects.rb

```diff
@@ -421,10 +421,12 @@ def create_fork(id, options={})
     #   Gitlab.edit_project(42, { name: 'project_name' })
     #
     # @param  [Integer, String] project The ID or name of a project.
-    # @param  [Hash] options A customizable set of options.
+    # @param  [Hash] options A customizable set of options
     # @option options [String] :name The name of a project
-    # @option options [String] :path The name of a project
-    # @option options [String] :description The name of a project
+    # @option options [String] :path The URL-safe path of the project (used in Gitlab URLs)
```

Comment:
Actually, on second thought... it would probably be safer to just pass the user-provided value through unchanged, to ensure they're getting the path they wanted (or an error if it's not valid).  If they want an automatic option, the path is auto-generated from the name, so they don't need to pass a path at all.  I'll update the docs to reflect that.